config/default.json5

```diff
@@ -10,6 +10,17 @@
   },
 
   blockchain: {
+    // Topics that will be listened to
+    topics: [
```

Comment:
Do we actually need that here?

Reply:
As I see, cache don't have the support of topics filtering, as we had in pinner

Reply:
lol you are right, the porting PR is still to be merged :-D I will merge it then :-)

Reply:
=)